src/store/store.js

```diff
@@ -0,0 +1,50 @@
+import Vue from "vue"
+import Vuex from "vuex"
+
+import { getData } from "../api/apiCalls"
+
+Vue.use(Vuex)
+
+export const store = new Vuex.Store({
+  state: {
+    shopItems: [],
+  },
+  mutations: {
+    getShopItems(state, payload) {
+      state.shopItems = payload
+    },
+    deleteShopItem(state, payload) {
```

Comment:
Consider renaming `payload` to better match what is the expected parameter.  
Maybe `itemToDeleteId` would work better?